.travis.yml

```diff
@@ -36,5 +36,8 @@ matrix:
         - env: CI_TASK=build CI_CONFIG=Debug
           os: osx
           osx_image: xcode11
+        - env: CI_TASK=build CI_CONFIG=Release CI_ZKSNARK=PGHR16 CI_USE_DOCKER=1
```

Comment:
`PGHR16` -> PGHR13

Reply:
Thanks!  Actually, that should have caused an error in cmake on travis.  Turns out I need to explitily pass the envvar to docker.  Added that as well.

Reply:
Yes was surprised to see the build passing